modules/standard/Random.chpl

```diff
@@ -135,13 +288,15 @@ class RandomStream {
     return result;
   }
 
-  proc fillRandom(X: [], param parSafe = this.parSafe) {
-    if X.eltType != complex && X.eltType != real && X.eltType != imag then
+  pragma "no doc"
+  proc fillRandom(arr: [], param parSafe = this.parSafe) {
```

Comment:
So we only intend for the fillRandom function to be used, not the fillRandom method on the RandomStream type?